src/com/palmergames/bukkit/towny/command/ResidentCommand.java

```diff
@@ -464,7 +464,7 @@ else if (resident.isMayor())
 				formatedList.add(colour + resident.getName() + Colors.White);
 			}
 		}
-		for (String line : ChatTools.list(formatedList))
+		for (String line : formatedList)

```

Comment:
Three changes to this file should probably be reverted, otherwise we're not getting a nice "name, name, name" list.